code/edgeidentity/src/main/java/com/adobe/marketing/mobile/edge/identity/IdentityItem.java

```diff
@@ -47,37 +50,41 @@ public IdentityItem(final String id, final AuthenticatedState authenticatedState
 
 	/**
 	 * Creates a new {@link IdentityItem} with default values
-	 * authenticatedState is set to AMBIGUOUS
-	 * primary is set to false
+	 * {@code authenticatedState) is set to AMBIGUOUS
+	 * (@code primary} is set to false
 	 * @param id the id for this {@link IdentityItem}
 	 */
 	public IdentityItem(final String id) {
 		this(id, AuthenticatedState.AMBIGUOUS, false);
 	}
 
 	/**
-	 * Creates a copy of item
+	 * Creates a copy of item.
 	 * @param item A {@link IdentityItem} to be copied
 	 */
 	public IdentityItem(final IdentityItem item) {
 		this(item.id, item.authenticatedState, item.primary);
 	}
 
 	/**
+	 * Identity of the consumer in the related namespace.
 	 * @return The id for this identity item
 	 */
 	public String getId() {
 		return id;
 	}
 
 	/**
+	 * The state this identity is authenticated.
 	 * @return Current {@link AuthenticatedState} for this item
 	 */
 	public AuthenticatedState getAuthenticatedState() {
 		return authenticatedState;
 	}
 
 	/**
+	 * Indicates this identity is the preferred identity.
```

Comment:
Can we reword to : `Indicates if this identity is the preferred identity or not.` ?
 
Also, should we give a hint on this that there can only be one active primary identity at any point in time?

Reply:
I'll change to `Indicates if this identity is the preferred identity`. 
I would prefer to link to external docs on how to use identifiers and the IdentityMap.  I don't think this API is the right place to detail how an XDM schema should be created and used. However, a quick search of the Identity Service documentation doesn't give any description on how to use the `primary` field. https://experienceleague.adobe.com/docs/experience-platform/identity/home.html?lang=en

Reply:
One thing I am concerned with that if the link changes our API docs will be incorrect (unless the sdk is updated and the customer does an sdk update too), so I would prefer to still keep this in our docs where we do updates more often.